src/index.ts

```diff
@@ -24,9 +24,14 @@ handlersMap.set('ping', new PingCommand(pingCmd));
 handlersMap.set('traceroute', new TracerouteCommand(traceCmd));
 handlersMap.set('dns', new DnsCommand(dnsCmd));
 
-logger.info(`Start probe version ${VERSION} in a ${process.env['NODE_ENV'] ?? 'production'} mode`);
+logger.info(`Start probe version ${VERSION} in a ${processTS.env['NODE_ENV'] ?? 'production'} mode`);
 
 function connect() {
+	const worker = {
+		jobs: new Map<string, number>(),
```

Comment:
Do we really need a map of jobs? I can't find where the timestamp we added to the map is used. A simple incrementor should work here as well.

Reply:
how else do you know how many jobs are remaining? The timestamp is there just to fill in the blank; it could've been an array of job Id's as well - the important part is having a list of uncompleted jobs.

Reply:
timestamp will also help to kill the job if it takes too long - just an idea, not implemented.

Reply:
>how else do you know how many jobs are remaining?

`worker.activeJobs++` and `worker.activeJobs--`

Reply:
I feel like a map of IDs is better suited here. If we already track jobs, we could as well keep their references.